examples/systemcalls/systemcalls.c

```diff
@@ -16,8 +21,24 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
+   int retVal = 0;
+
+   /*trigger the system call*/
+   retVal = system(cmd);
+   
+   /*if a child process couldnt be created*/
+   if(-1 == retVal)
```

Comment:
@AnanthD21 think about what would happen if the cmd returned with an error code other than -1 here.